app/uk/gov/hmrc/tariffclassificationfrontend/models/SampleReturn.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.models
+
+object SampleReturn extends Enumeration {
+  type SampleReturn = Value
+
+  val YES, NO, TO_BE_CONFIRMED = Value
+
+  def format(status: Option[SampleReturn]): String = {
+    status match {
+      case Some(YES) => "Yes"
+      case Some(NO) => "No"
+      case Some(TO_BE_CONFIRMED) => "To be confirmed"
+      case _ => "Unknown"
```

Comment:
Was just wondering with this - should case of None just be 'To be confirmed'? Still makes sense, but possibly looks  bit more pleasant than Unknown

Reply:
Should never happen.  If the status is None then it shouldn't be rendered.

Reply:
should never happen = thrown exception ?